src/SPC/builder/linux/LinuxBuilder.php

```diff
@@ -278,7 +278,7 @@ protected function buildMicro(): void
      */
     protected function buildFpm(): void
     {
-        $vars = SystemUtil::makeEnvVarString($this->getMakeExtraVars());
```

Comment:
makeEnvVarString is different from `getEnvString`. `getEnvString` is more like a environment dumper, and `makeEnvVarString` is to generate k-v array to string.

Reply:
Sorry, that was entirely unintentional